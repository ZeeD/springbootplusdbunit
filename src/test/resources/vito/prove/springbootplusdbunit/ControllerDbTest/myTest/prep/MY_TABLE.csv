id,name,when
123,foo1,2020-06-22 15:33:00
456,foo2,null
789,,1234-05-06 07:08:09
