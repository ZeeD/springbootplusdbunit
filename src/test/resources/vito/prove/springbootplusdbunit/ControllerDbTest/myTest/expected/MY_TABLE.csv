id,name,when
1,name1,1970-01-01 01:01:40
123,"foo1",2020-06-22 15:33:00
456,"foo2",null
789,null,1234-05-06 07:08:09
